src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -59,18 +69,27 @@ public void launchTask(final ExecutorDriver driver, final TaskInfo taskInfo) {
           .setTaskId(journalNodeTask.taskInfo.getTaskId())
           .setState(TaskState.TASK_RUNNING)
           .build());
+      healthCheckJN = new TimedHealthCheck(driver, journalNodeTask);
+      timer.scheduleAtFixedRate(healthCheckJN, schedulerConf.getHealthCheckWaitingPeriod(),
+          schedulerConf.getHealthCheckFrequency());
```

Comment:
This supports http and command only which is not exactly what we want here. We may want to add an http health check and we could use that API for that when we improve our health check.